### PR TITLE
feat: add xdebug_info (fixes #8977)

### LINKED ISSUE #8977: xdebug_info triggers UndefinedFunction
See https://xdebug.org/docs/all_functions#xdebug_info

https://psalm.dev/r/1a65c240f2

Another xdebug method like `xdebug_get_stack_depth` does not trigger an UndefinedFunction: https://psalm.dev/r/4646f18e42

### PR DESCRIPTION
**Pull request by mathe42**
This should fix #8977.

This is my first contribution (just found psalm a few days ago) so feel free to request changes as needed...

### CODE REVIEW

File: dictionaries/CallMap_historical.php [hunk 1]
```diff
@@ -16383,6 +16383,7 @@
     'xdebug_get_profiler_filename' => ['string|false'],
     'xdebug_get_stack_depth' => ['int'],
     'xdebug_get_tracefile_name' => ['string'],
+    'xdebug_info' => ['mixed', 'category='=>'string'],
```
Comment (weirdan) [inline dictionaries/CallMap_historical.php:16386]: `xdebug_info` was introduced in Xdebug 3. Xdebug 3 is only compatible with PHP 7.2 (https://xdebug.org/docs/compat#versions), and `CallMap_historical` documents the available functions for PHP 7.0. So I'd suggest moving `xdebug_info` from here to `CallMap_72_delta`.

**Review by weirdan — CHANGES_REQUESTED** [review]
Congrats on your first PR!

There are some changes needed to get this merged; please see below.

File: dictionaries/CallMap_historical.php (hunk 1 above)
Comment (mathe42) [reply dictionaries/CallMap_historical.php]: changed that!

**Review by weirdan — APPROVED** [review]

**Comment by orklah** [comment]
Thanks!

**Pull request merged**